src/pages/hyunjaeKim/Login/Login.js

```diff
@@ -1,8 +1,80 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Login.scss";
 
 class LoginHyunJae extends Component {
+  state = {
+    email: "",
+    password: "",
+  };
+
+  saveData = data => {
+    localStorage.setItem("TOKEN", data.TOKEN);
+  };
+
+  handleLogin = () => {
+    fetch("http://10.58.0.45:8000/users/login", {
+      method: "POST",
+      body: JSON.stringify({
+        ...this.state,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.TOKEN) {
+          this.saveData(data);
+          this.props.history.push("/main-jae");
+        } else {
+          alert("회원가입을 해주세요!");
+        }
+      });
+  };
+
+  handleIDInput = e => {
+    const { value } = e.target;
+    this.setState({ ...this.state, email: value });
+  };
+
+  handlePWInput = e => {
+    const { value } = e.target;
+    this.setState({ ...this.state, password: value });
+  };
+
   render() {
-    return <div>안녕ㅎㅏ세요?</div>;
+    return (
+      <div className="loginContainer">
+        <h1 className="title" aria-label="Welcome to Westagram">
+          Westagram
+        </h1>
+        <form className="userInfo">
+          <input
+            type="email"
+            className="userID"
+            placeholder="이메일을 입력해주세요"
+            aria-label="Write account ID"
+            onChange={this.handleIDInput}
+          />
+          <input
+            type="password"
+            className="userPW"
+            placeholder="비밀번호"
+            aria-label="Write accout password"
+            onChange={this.handlePWInput}
+          />
+          <button
+            type="button"
+            className={this.state.isActive ? "loginBtn active" : "loginBtn"}
```

Comment:
```suggestion
            className={`loginBtn ${isActive ? 'active' : ''}`}
```
구조분해 할당과 템플릿 리터럴을 활용하면 위와 같은 표현도 가능합니다. 고정된 요소만 놔두고 변화하는 요소만 바꿔주면 중복을 피할 수 있겠네요.

Reply:
넵! 반영하여 변경하였습니다:)